phpcs.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0"?>
+<ruleset name="FunFunFundraisingFrontend">
```

Comment:
Could you use https://github.com/JeroenDeDauw/new-php-project/blob/master/phpcs.xml instead? It's basically the same, only with a more appropriate name.

Reply:
I have copied over the file from there now. I want to note that I had to remove the mediawiki code sniffs since there is another Composer dependency incompatibility which prevents the `mediawiki/mediawiki-codesniffer` dependency from being installed.